src/main/java/com/ustudent/resquod/exception/PositionAlreadyExistsException.java

```diff
@@ -0,0 +1,4 @@
+package com.ustudent.resquod.exception;
+
+public class PositionAlreadyExistsException extends Exception {
```

Comment:
Wyjątki dziedziczą po RuntimeException raczej

Reply:
Raczej to powinno się używać wyjątków typu checked czyli właśnie tych dziedziczących po Exception